specs-go/oci.go

```diff
@@ -102,3 +100,14 @@ func toOCIDevice(d *DeviceNode) spec.Mount {
 		Options:     d.Permissions,
 	}
 }
+
+func toOCILinuxDevice(d *DeviceNode) spec.LinuxDevice {
+	return spec.LinuxDevice{
+		Path:  d.ContainerPath,
+		Type:  d.Type,
+		Major: d.Major,
+		Minor: d.Minor,
+		UID:   d.UID,
+		GID:   d.GID,
```

Comment:
We don't set `FileMode` based on the contents of permission here. I assume that this will be addressed in a follow-up PR.

Reply:
correct